### ISSUE TITLE
Question: Is this tool less apt to have banned accounts than the other websites?

### ISSUE THREAD

**Issue by Psykoflo** [Wolfsblvt/BlossomsPokemonGoManager#287]
Hi!
I just discovered your tool and it's great, thanks a lot :)
Just a quick question: as this tool goes through our PC, and so our Wifi connection (and so hopefully our real location), does Niantic can also "flag" our account as we are using a third party tool? Just like he is doing for using PGnexus our Pogobag and ban accounts?
Or is it safer to use?
What location is given to Niantic when we connect through it? I guess that if it's our real location, then Niantic cannot identify that we are using a third party tool right?
Thanks, I hope I'm in the good section for questions :)
And congrats again for this great development!

**Comment by Wolfsblvt** [comment]
It's difficult to have a real answer to most of that.
We can't look into Niantics heads and see what they do, will do and can do.

But I can try to post my view on this. First: Our tool is just a small one. We do not have any real botting functionality, we do not use any of those calls that got secured by unknown6 and stuff by Niantic. So we aren't considered a "high risk".
Do we violate the ToS? Yes, probably. Like we have explained in the Disclaimer.
Can Niantic see that you use a Third-Party-Tool if they want? Probably. I think there is no way to really hide that. Can they easily get that with automated mechanisms? Maybe. But most likely not. We try to do as less calls as possible and look as less like a machine as possible.
We have added custom delays for bulk operations now, we do not query any real safe data, etc etc.

For location: It's a bit different. We do not send any location to the API. That means the server never receives where you are. It's like if you use your phone with GPS deactivated or when you are somwhere inside and it can't get your location. Guess that's the safest way how such a tool can handle location.

The most important question: **Will Niantic ban you for using this tool?**
Probably not. Sure they can, if they want to. But if they are going to ban every person that uses IV checkers and similar stuff, they loose a big part of their players. Guess most important issues for them are bots, that gather Pokémon, turn stops and fight gyms. We do neither of that.

So far, there is **no known case of a person getting banned for using BGPM**. That's something I can't point out enough. And I guess that's what the current situation is.
